File: dispatch/src/main/java/com/epro/dispatch/MainActivity.java
package com.epro.dispatch;

import android.content.Intent;
import android.os.Bundle;

import com.blankj.utilcode.util.Utils;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        startActivity(new Intent(Utils.getApp(), DispatchCustomBtnActivity.class));
 //       startActivity(new Intent(Utils.getApp(), DispatchActivity.class));

    }
}

File: filepicker/src/main/java/com/epro/filepicker/MainActivity.kt
package com.epro.filepicker

import android.content.Intent
import android.content.Intent.EXTRA_ALLOW_MULTIPLE
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.blankj.utilcode.util.AppUtils
import com.blankj.utilcode.util.GsonUtils
import com.blankj.utilcode.util.LogUtils
import com.blankj.utilcode.util.Utils
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    companion object{
        const val REQUEST_CODE = 0X11
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        tv_open_sys_file_picker.setOnClickListener {
            var intent = Intent(Intent.ACTION_GET_CONTENT)
            intent.setType("image/*")
            //下面代码可以多选 图片阿实用
//            intent.putExtra(EXTRA_ALLOW_MULTIPLE, true)
            startActivityForResult(intent, REQUEST_CODE)
        }
    }

    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
        super.onActivityResult(requestCode, resultCode, data)
        LogUtils.e(GsonUtils.toJson(data))
    }
}
